tests/PluginTest.php

```diff
@@ -128,13 +128,12 @@ protected function composerJSONDefaults() {
       ),
       'require' => array(
         'drupal-composer/drupal-scaffold' => $this->tmpReleaseTag,
-        'composer/installers' => '^1.0.20',
-        'drupal/core' => '8.0.0',
+        'composer/installers' => '^1.0.24',
+        'drupal/core' => '8.1.0-rc1',
       ),
       'scripts' => array(
         'drupal-scaffold' =>  'DrupalComposer\\DrupalScaffold\\Plugin::scaffold'
-      ),
-      'minimum-stability' => 'dev',
```

Comment:
The minimum-stability flag is required. The test can't resolve ``'drupal/core' => '8.1.0-rc1'`` otherwise.